miden/src/repl/mod.rs

```diff
@@ -251,7 +251,7 @@ pub fn start_repl(library_paths: &Vec<PathBuf>, use_stdlib: bool) {
                             }
                             // in case the flag has not been initialized.
                             if !mem_at_addr_present {
-                                println!("Memory at address {} is empty", addr);
+                                println!("Memory at address {:?} is empty", addr);
```

Comment:
We should implement `Display` for `MemoryAddress` here instead of using the `Debug` version. For the `Display` implementation, we can just forward to the `u32` implementation

Reply:
Done.